src/syntax/syntax.rkt

```diff
@@ -342,9 +331,17 @@
 
 (define-operator (fma real real real) real [spec (lambda (x y z) (+ (* x y) z))])
 
-(define-operator (erfc real) real [spec (lambda (x) (- 1 (erf x)))])
-
 (module+ test
+  ; check expected number of operators
+  (check-equal? (length (all-operators)) 63)
+
+  ; check that Rival supports all non-accelerator operators
+  (for ([op (in-list (all-operators))] #:unless (operator-accelerator? op))
+    (define vars (map (lambda (_) (gensym)) (operator-info op 'itype)))
+    (define disc (discretization 64 #f #f)) ; fake arguments
+    (rival-compile (list `(,op ,@vars)) vars (list disc)))
```

Comment:
This is a neat check but I should probably add a `rival-expr?` test to Rival and then use that here. Anyway, not this PR probably.

Reply:
Yeah Rival should have a way to test